### PR TITLE
Add unit tests for standard output truth csv

### PR DESCRIPTION
**Pull request by Khoale1096**

### CODE REVIEW

**Review by elray1 — CHANGES_REQUESTED** [review]
I got a couple of failed tests here, and had a question about handling of expected outputs for DC.  Thanks!

File: tests/testthat/test-get_truth.R [hunk 1]
```diff
@@ -4,6 +4,147 @@ library(covidData)
...
+  expected_column_names <- c('date','location','location_name','value')
+  
+  expect_equal(actual_cumulative_deaths_column_names, expected_column_names)
+  expect_equal(actual_incident_deaths_column_names, expected_column_names)
+  expect_equal(actual_cumulative_cases_column_names, expected_column_names)
+  expect_equal(actual_incident_cases_column_names, expected_column_names)
+})
+
+
+test_that("preprocess_jhu files has expected combinations of location, week", {
```
[the remark this answers is not in the document]
Comment (Khoale1096) [reply tests/testthat/test-get_truth.R:26]: Sorry it's my bad. I have a commit that already resolved this but didn't push it through

File: tests/testthat/test-get_truth.R [hunk 2]
```diff
@@ -4,6 +4,147 @@ library(covidData)
...
+    dplyr::select(date, location)
+  
+  actual_incident_deaths <- actual$incident_deaths %>%
+    dplyr::select(date, location)
+  
+  expected_deaths <- covidData::load_jhu_data(spatial_resolution = spatial_resolutions,
+                                              temporal_resolution = "daily",
+                                              measure = "deaths")
+  
+  expected_cumulative_deaths <- expected[,c("date", "location", "cum")] %>%
```
[the remark this answers is not in the document]
Comment (Khoale1096) [reply tests/testthat/test-get_truth.R]: Sorry it's my bad. I have a commit that already resolved this but didn't push it through

**Comment by Khoale1096** [comment]
@elray1 I have compared the outputs of these new methods with the output of the python scripts. There is only one discrepancy, which is that the incident cases/deaths output of the python scripts do not have any negative values due to thresholding at this [code](https://github.com/reichlab/covid19-forecast-hub/blob/master/data-truth/truth_utils/_utils.py#L155). However, we decided [here](https://reichlab.slack.com/archives/C011SKXLBA8/p1607106472289400?thread_ts=1606752765.242800&cid=C011SKXLBA8) that we would like to not truncate these negative values. So I removed the thresholding code on the python side and the output of the two codes match. Is there any other check I should do before merging this in?

**Pull request merged**